unethical-autologin/src/main/java/net/unethicalite/plugins/autologin/UnethicalAutoLoginPlugin.java

```diff
@@ -152,13 +152,30 @@ private void login()
 	{
 		client.setUsername(config.username());
 		client.setPassword(config.password());
-
-		Mouse.click(299, 322, true);
+		Time.sleep(1000);
+		Keyboard.sendEnter();
+		Keyboard.sendEnter();
+		Keyboard.sendEnter();
 	}
 
 	private void enterAuth()
 	{
 		client.setOtp(new Totp(config.auth()).now());
 		Keyboard.sendEnter();
 	}
+
+	@Override
+	protected int loop()
+	{
+		Time.sleep(60_000);
+		if (client.getGameState() == GameState.LOGIN_SCREEN)
+		{
+			if (client.getLoginIndex() == 2) login();
```

Comment:
whats the 2 here? I assume you found it through inspection but I'm curious if you have any insight as to what values this index can be. 

Also, any protests to changing this to just be one `if` with an `&&`?

Reply:
0 is the welcome screen, 2 is logging in screen. There are few others that you can check in the LoginEvent.